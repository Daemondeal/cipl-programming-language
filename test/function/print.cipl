proc foo():
    pass

print(foo)
print(print)