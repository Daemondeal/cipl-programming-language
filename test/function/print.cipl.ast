(proc foo (body
    pass
)
(expression (call print foo))
(expression (call print print))

