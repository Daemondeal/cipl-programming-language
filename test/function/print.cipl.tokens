Token          Lexeme         Line 
PROC           proc           1    
IDENTIFIER     foo            1    
LEFT_PAREN     (              1    
RIGHT_PAREN    )              1    
COLON          :              1    
LINE_END       \n             2    
INDENT                        2    
PASS           pass           2    
LINE_END       \n             3    
DEDENT                        4    
IDENTIFIER     println        4    
LEFT_PAREN     (              4    
IDENTIFIER     foo            4    
RIGHT_PAREN    )              4    
LINE_END       \n             5    
IDENTIFIER     println        5    
LEFT_PAREN     (              5    
IDENTIFIER     println        5    
RIGHT_PAREN    )              5    
LINE_END       \n             5    
EOF                           5    
