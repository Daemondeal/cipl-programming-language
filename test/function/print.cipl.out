::stdout::
<proc foo>
<native proc>

::stderr::
